library/std/src/sync/barrier.rs

```diff
@@ -31,6 +33,9 @@ pub struct Barrier {
     num_threads: usize,
 }
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Not that it matters much, but the original `RefUnwindSafe` implementation was later added in 1.12.0.
```suggestion
#[stable(feature = "unwind_safe_lock_refs", since = "1.12.0")]
```